crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -555,6 +555,7 @@ module sui_system::sui_system {
         ctx: &mut TxContext,
     ) : Balance<SUI> {
         let self = load_system_state_mut(wrapper);
+
```

Comment:
nit spurious change makes source verification potentially sad (though not an issue since you are making other changes, just noting for the future)

Reply:
my bad, reverting